spec.html

```diff
@@ -26843,7 +26832,7 @@ <h1>Cyclic Module Records</h1>
                 an integer or ~empty~
               </td>
               <td>
-                Auxiliary field used during Link and Evaluate only. If [[Status]] is either ~linking~ or ~evaluating~, this is either the module's own [[DFSIndex]] or that of an "earlier" module in the same strongly connected component.
+                Auxiliary field used during Link and Evaluate only. If [[Status]] is either ~linking~ or ~evaluating~, this is the lowest depth-first index of a module in this module's strongly connected component that has been visited so far.
```

Comment:
This is a stronger claim than previously: the old text only claims it was _some_ previous value; now it's specifically the lowest. Is the stronger claim actually true? At a glance it's a little unclear, because transitions of DFSAncestorIndex happen imperatively in InnerModuleLinking rather than happening all at once.

Reply:
Mh true, I need to reword it. This value starts with the module's index, and progressively gets lower.